# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2005/02/08 12:35:49-02:00 marcelo@logos.cnet 
#   Solar Designer: missing f_maxcount initialization
#   TAG: MailDone
# 
# fs/nfsd/vfs.c
#   2005/02/08 12:35:28-02:00 marcelo@logos.cnet +2 -0
#   Solar Designer: missing f_maxcount initialization
# 
# fs/file_table.c
#   2005/02/08 12:34:05-02:00 marcelo@logos.cnet +2 -0
#   Solar Designer: missing f_maxcount initialization
# 
diff -Nru a/fs/file_table.c b/fs/file_table.c
--- a/fs/file_table.c	2005-02-08 13:03:17 -08:00
+++ b/fs/file_table.c	2005-02-08 13:03:17 -08:00
@@ -92,6 +92,8 @@
 	filp->f_uid    = current->fsuid;
 	filp->f_gid    = current->fsgid;
 	filp->f_op     = dentry->d_inode->i_fop;
+	filp->f_maxcount = INT_MAX;
+
 	if (filp->f_op->open)
 		return filp->f_op->open(dentry->d_inode, filp);
 	else
diff -Nru a/fs/nfsd/vfs.c b/fs/nfsd/vfs.c
--- a/fs/nfsd/vfs.c	2005-02-08 13:03:17 -08:00
+++ b/fs/nfsd/vfs.c	2005-02-08 13:03:17 -08:00
@@ -466,6 +466,8 @@
 	atomic_set(&filp->f_count, 1);
 	filp->f_dentry = dentry;
 	filp->f_vfsmnt = fhp->fh_export->ex_mnt;
+	filp->f_maxcount = INT_MAX;
+
 	if (access & MAY_WRITE) {
 		filp->f_flags = O_WRONLY|O_LARGEFILE;
 		filp->f_mode  = FMODE_WRITE;
